components/catalogs/back/tests/Integration/Infrastructure/Controller/Internal/UpdateCatalogActionTest.php

```diff
@@ -55,6 +55,9 @@ public function testItUpdatesTheCatalog(): void
                         'value' => true,
                     ],
                 ],
+                'product_value_filters' => [
+                    'channel' => ['ecommerce'],
```

Comment:
```suggestion
                    'channels' => ['ecommerce'],
```

with an `s` in test files too